setup.cfg

```
@@ -159,7 +159,7 @@ exclude =
     doc/source/getting_started/comparison/includes/*.rst
 
 [codespell]
-ignore-words-list = ba,blocs,coo,hist,nd,sav,ser
+ignore-words-list = ba,blocs,coo,hist,nd,sav,ser,ines
 ignore-regex = https://(\w+\.)+
```

this could potentially mask typos, we'd want to keep `ignore-words-list` as short as possible

would this
```suggestion
ignore-words-list = ba,blocs,coo,hist,nd,sav,ser
ignore-regex = https://([\w/\.])+
```
work?